habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -137,6 +137,14 @@ class BaseVelocityActionConfig(ActionConfig):
     ang_speed: float = 10.0
     allow_dyn_slide: bool = True
     allow_back: bool = True
+    collision_threshold: float = 1e-5
+    navmesh_offset: Optional[List[float]] = None
+    min_displacement: float = 0.05  # minimum displacement that is allowed
+    max_displacement_along_axis: float = 0.5  # maximum displacement
```

Comment:
@ykarmesh do these values look good - do you think they take into account the resolution of the controller?

Reply:
It really depends on how the real controller behaves. Lets ask Chris.